tests/Mixins/StoreCollectionTest.php

```diff
@@ -38,4 +39,26 @@ public function can_store_a_collection_as_excel_on_non_default_disk()
         $this->assertTrue($response);
         $this->assertFileExists(__DIR__ . '/../Data/Disks/Test/collection-store.xlsx');
```

Comment:
You added the following assertion for withHeading=true, but the current test doesn't validate if the content of the exported file is correct. My current assumption is that the first row will be empty now, which would mean this would be a breaking change :( Can you please verify this.

```
if (is_file($file)) {
            $array = $this->readAsArray($file, Excel::XLSX);
            $this->assertEquals($collection->collapse()->keys()->all(), collect($array)->first());
        }
```

Reply:
it's done, but the other tests don't check the content either, perhaps it's something to be done later on.

Reply:
it's done, but the other tests don't check the content either, perhaps it's something to be done later on.

Reply:
That’s my point. We need to add that contrnt test to protect against regression. We cannot do it later on as this PR might cause a breaking change. I can only merge if I’m sure it’s not a BC.

Reply:
yes, i understand, i have added tests to check that it works with or without headers and that the content and the headers match. When i'm saying later on, it's for the tests already present that i prefer to left untouched for now.

Reply:
I see it’s added now.